enterprise-architecture/update.sh

```diff
@@ -10,7 +10,7 @@ git fetch origin
 git checkout origin/master
 git checkout -b $BRANCH
 
-curl https://ea.gsa.gov/api/v0/itstandards | python3 json_to_csv.py > it-standards.csv
+python3 fetch.py > it-standards.csv
```

Comment:
Fun fact: macOS Mojave ships with Bash 3, which doesn't seem to handle `pipefail`s properly 🙃 Moved the downloading to Python so it works more reliably.